core/urls.py

```diff
@@ -20,5 +20,5 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/v1/auth/", include("users.urls")),
+    path("api/v1/", include("users.urls")),
```

Comment:
Maybe you can store all your apps in one folder called app then put users app and any other subsequent apps, and point this url to the urls.py in app, then in the app/urls.py you can paths for each individual app?

Reply:
We decided not to put our apps in a single folder for this project